magma/bits.py

```diff
@@ -503,6 +503,14 @@ def as_bits(self):
     def from_bits(cls, value):
         return value
 
+    def __getitem__(self, index):
```

Comment:
is this an extraneous change?

Reply:
This was necessary for the test

Reply:
The issue was with this code
```
    class TestDefault:
         def __init__(self):
             self.x: m.Bits[8] = m.bits(0xFE, 8)

         def __call__(self, index: m.UInt[3]) -> m.Bit:
             return self.x[index]
```
this revealed that get item wasn't implemented.

Reply:
ok, just so I understand, the dynamic select as part of `__getitem__`  was part of the spec, just dropped when we moved to the new types? (or is it something we're adding to the spec?)

Reply:
yes, it was previously implemented inside mantle (which I think it worked originally, but magma doesn't import mantle to patch on the implementation)